src/mgos_ili9341.c

```diff
@@ -525,8 +525,9 @@ void mgos_ili9341_drawDIF(uint16_t x0, uint16_t y0, char *fn) {
   int       fd;
 
   fd = open(fn, O_RDONLY);
-  if (!fd) {
-    LOG(LL_ERROR, ("%s: Could not opens", fn));
+   // has to be tested not only for NULL
+  if (fd <= 0) {
```

Comment:
Great catch, thanks! 
One nit - open() can return 0 as a valid file descriptor. Errors are returned as -1:
       open(), openat(), and creat() return the new file descriptor, or -1 if an error occurred (in which case, errno is set appropriately).

Can you make this "if (fd < 0)" please ?

Reply:
Hm, I read two "flavours": one said _"Returns the file descriptor for the new file. The file descriptor returned is always the smallest integer greater than zero that is still available. If a negative value is returned, then there was an error opening the file."_, the other states _"... return a non-negative integer representing the lowest numbered unused file descriptor."_ So I'm a little confused, and in my experience a file descriptor of "0" doesn't "taste" right ... ;-) But ok, I'll try to change that.

Reply:
Now I changed it and did amend my commit ... I presume no new PR will be necessary?

Reply:
Thanks! FYI, fd=0 is stdin, fd=1 is stdout, fd=2 is stderr usually in unix processes.
See https://en.wikipedia.org/wiki/File_descriptor

No further action necessary, merging now!

Reply:
Well, some years passed since I worked on a Unix System V (a real one!) ;-) and was aware of the paradigm "everything is a file". So stdout is 1 and stderr is 2 - yeah, right, I remember ... use it on the bash very often ... But I have the luck to always stumble upon some docs used as memory refreshing which are stating nonsense ...